src/main/java/calculator/CalculationText.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CalculationText {
+
+    private static String delimeter = ",|:";
+    private static final String delimeterDividor = "|";
+    private static Pattern calculatorMatcher = Pattern.compile("//(.)\n(.*)");
+    private static String[] DEFAULT_VALUE = { "0" };
+    private String[] calculationText;
```

Comment:
상수와 변수를 문맥상 분리하기 위해 개행을 하나 넣어줍시다:)
```suggestion

    private String[] calculationText;
```